### PR TITLE
Input radio saving - Issue #148

### PR DESCRIPTION
**Pull request by Kreyco**
PR gets a solution to issue #148 , and it also gives the best way to save input radio.
Because it erases the unchecked input radio.

### CODE REVIEW

File: .idea/modules.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (simsalabim) [inline .idea/modules.xml]: Please remove unrelated files from the changeset.

**Comment by simsalabim** [comment]
Thanks for your contribution!

**Comment by Kreyco** [comment]
@simsalabim 'Remove unrelated files from the changeset' done. Now PR is ready to merge. __Thank you!__
